README

```diff
@@ -1,70 +0,0 @@
-=============================
-       Linux, OS-X, BSD
-=============================
-
-Start by executing:
-./bootstrap
-
-To install add-ons into /path/to/XBMC:
```

Comment:
I thought it was not very clear where exactly the addons will end up being installed. Is the idea to use e.g. `/usr/local/lib/kodi`?